tables/tables.go

```diff
@@ -192,13 +192,14 @@ var NamePriority = map[string]int{
 	"of":                -84, // for check_dependencies
 	"baseline":          -83, // for searchbox_library
 	// All others sort here, at 0.
-	"destdir":        1,
-	"exports":        2,
-	"runtime_deps":   3,
-	"deps":           4,
-	"implementation": 5,
-	"implements":     6,
-	"alwayslink":     7,
+	"destdir":              1,
+	"exports":              2,
+	"runtime_deps":         3,
+	"deps":                 4,
+	"implementation":       5,
+	"implements":           6,
+	"alwayslink":           7,
+	"//conditions:default": 50, // default condition is the last one by convention
```

Comment:
The other lines are mostly for keyword-arguments when calling a rule. Since this last line can be surprising I recommend updating the comment.

It should mention it's for the dictionary in select.